lib/views/enablenotification_screen.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+
+class EnableNotificationscreen extends StatelessWidget {
+  const EnableNotificationscreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: Padding(
+          padding: const EdgeInsets.all(16.0),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: <Widget>[
+              const Spacer(),
+              Image.asset(
+                'assets/images/notification.png',
+                width: 200.0,
+                height: 200.0,
+              ),
+              const SizedBox(height: 24),
+              const Text(
+                'Turn on Notifications ?',
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  fontSize: 24,
+                  fontWeight: FontWeight.bold,
+                  color: Colors.black,
+                ),
+              ),
+              const SizedBox(height: 8),
+              const Text(
+                "Receive instant updates on live lectures, upcoming quizzes, and important announcements. Never miss a beat in your academic journey.",
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  fontSize: 16,
+                  color: Colors.black54,
+                ),
+              ),
+              const SizedBox(height: 48),
+              const Spacer(),
+              ElevatedButton(
+                style: ElevatedButton.styleFrom(
```

Comment:
Add this to a theme 😄

Reply:
done ✅  see this [commit](https://github.com/TUM-Dev/gocast_mobile/commit/e32ce3fa01ea8b05f6013cd2b31d9bce6da0abdd#diff-7eb87a0c9553264ec1d72f99b88c47c68e4be65ff1ddeccf4d09b8da09a28980)

p.s: Theme will be improved